### DISCUSSION TITLE
Compatibility issue with Woocommerce Custom Product Addons from Acowebs

### DISCUSSION THREAD

**Discussion by alzajac** [woocommerce/woocommerce#45077] [WooCommerce, Blocks, and Site Editing]
Hello, I noticed that there is some compatibility issue with WooCommerce Blocks and Woocommerce Custom Product Addons from Acowebs. I am using this plugin to add variation products but the options of this variations are showing not properly in Cart and Checkout pages made by WooCommerce Blocks (on standard WooCommerce cart and checkout pages everything is good). 

![Cart - WooCommerce Blocks](https://user-images.githubusercontent.com/58148038/118468879-7c1e1d80-b705-11eb-87c3-36322b64c129.png)

![Checkout - WooCommerce Blocks](https://user-images.githubusercontent.com/58148038/118468893-804a3b00-b705-11eb-8f54-5aef583e59e3.png)

**Comment by senadir** [comment]
Hey, we're not fully compatible with all plugins, so issues like this will pop up.

For the issue you're mentioning, it can be fixed by change on Acowebs side.

When defining the meta field, they should set `__experimental_woocommerce_blocks_hidden` attribute to `true` for it to be hidden on Cart and Checkout. Unfourtantly, I can't confirm this fix because I don't have access to that plugin.

I'm going to close this issue because there's nothing we can do here.
